src/neuroglancer/sliceview/decode_jpeg_stack.ts

```diff
@@ -20,14 +20,41 @@ import {Vec3, vec3} from 'neuroglancer/util/geom';
 export function decodeJpegStack(data: Uint8Array, chunkDataSize: Vec3) {
```

Comment:
Good call. I modified both files and added an error message for differing number of components and an unexpected number of components.